department-of-veterans-affairs/va.gov-cms
# VACMS-4320: Hours-Field-Vet-Center

## Description

See _issueid_. 

https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/4320

## Testing done


## Screenshots


## QA steps

- Navigate to preview environment:
- Navigate here: `/node/add/vet_center`
- [ ] Confirm the new Hours fields look like the following

![image](https://user-images.githubusercontent.com/56809719/109895488-49639d80-7c5d-11eb-950c-6104758143e4.png)

- Fill out all required fields and be sure the hours field matches the image below:

![image](https://user-images.githubusercontent.com/56809719/109895663-98a9ce00-7c5d-11eb-8d13-247045538f60.png)

- Save the node

- [ ] Confirm the following on node preview for the hours field:

![image](https://user-images.githubusercontent.com/56809719/109895779-d3136b00-7c5d-11eb-868f-4fb663c73d15.png)

- [ ] Repeat the same steps above for the following:
- [ ] `/node/add/vet_center_mobile_vet_center`
- [ ] `/node/add/vet_center_outstation`


### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
